nautobot/extras/tests/constants.py

```diff
@@ -0,0 +1,92 @@
+BIG_GRAPHQL_DEVICE_QUERY = """
+query ($device: [String!]) {
+  devices(name: $device) {
+    config_context
+    name
+    position
+    serial
+    primary_ip4 {
+      id
+      primary_ip4_for {
+        id
+        name
+      }
+    }
+    tenant {
+      name
+    }
+    tags {
+      name
+      slug
+    }
+    device_role {
+      name
+    }
+    platform {
+      name
+      slug
+      manufacturer {
+        name
+      }
+      napalm_driver
+      network_driver
+      network_driver_mappings
```

Comment:
Same question here about whether it's possible/desirable to drill down into specific keys under `network_driver_mappings` as I'd naively expect from GraphQL.